tempo_test.yml

```diff
@@ -0,0 +1,9 @@
+---
+- hosts: stats
+  become: true
+  vars_files:
+    - vars/common.yml
+  roles:
+    - name: tempo installation
+      role: tempo
+      tags: tempo
```

Comment:
I would suggest getting rid of this test playbook.

Reply:
https://github.com/ipfs-search/ipfs-search-deployment/commit/d83a8ed4e0c8886d348105f08a1d7b78892759d5